content/2015.09.kinto-encryption-example-fr.rst

```diff
@@ -0,0 +1,254 @@
+:summary: Comment utiliser Kinto.js pour chiffrer ses données dans Kinto.
+:slug: kinto-encryption-example
+:lang: fr
+:url: kinto-chiffrer-ses-donnees
+:scripts: kinto-61ad959.min.js, kinto-encryption-example.js
+
+.. note::
+
+    Traduit de l'anglais par `Rémy Hubscher <https://twitter.com/Natim>`_.
+
+Voici un article rédigé par `Michiel de Jong
+<https://michielbdejong.com/>`_ de l'équipe Firefox OS.
+
+Lorsqu'on souhaite sauvegarder les données d'une application
+"Offline-first" (qui fonctionne normalement hors connexion) de manière
+chiffrer dans le cloud, ça a du sens de les garder déchiffrées
+localement et de les chiffrer uniquement avant de les téléverser vers
+un serveur distant.
+
+Ainsi, il est facile et efficace d'y accéder pour les afficher ou
+mettre à jour la copie locale, sans avoir le coût répété de les
+chiffrer/déchiffrer à chaque opération.
+
+De la même manière, on peut déchiffrer les données venant du serveur
+une seule fois, tout de suite après les avoir télécharger et
+l'application peut ensuite afficher les données aussi souvent que
+nécessaire sans coût de calculs supplémentaires.
+
+Synchroniser les données d'une telle application vers et depuis "le
+cloud" est une science en elle-même et `Kinto.js
+<https://github.com/Kinto/kinto.js>`_ est un outil brillant pour ce
+fairem éveloppé par la toute aussi brillante équipe du Service de
+nuages de Mozilla. La release 1.0 ne devrait plus tarder et vous en
+saurez sûrement plus sur ce blog très prochainement.
+
+Il se trouve que Kinto,js supporte maintenant les `RemoteTransformers
+<http://kintojs.readthedocs.org/en/latest/api/#transformers>`_, qui
+permettent de manière très simple d'appliquer n'importe quel type de
+transformations que vous souhaitez faire sur les enregistrements d'une
+collection avant qu'ils ne soient stocker dans la base de données
+locale (``RemoteTransformer#decode``) mais aussi avant qu'ils ne
+soient sauvegarder sur le serveur distant
+(``RemoteTransformer#encode``).
+
+Cet article de blog montre comment utiliser un ``RemoteTransformer``
+pour chiffrer vos données juste avant de les envoyer dans les
+nuages. De plus nous utilisons WebCrypto pour la partie cryptographie.
+
+Notez que je ne suis pas un expert en sécurité et que ce code est
+simplement un exemple écrit en une après-midi. Si vous copiez/collez
+ce code, ne l'utilisez pas pour quoi que ce soit de sensible sans
+avoir fait votre propre revue de sécurité au préalable ! :) Par
+exemple il n'y a aucun mot de passe sur la clé de chiffrement, aucune
+signature et vérification des enregistrements, c'est uniquement un
+chiffrement/déchiffrement très basique pour la démonstration.
+
+L'idée derrière un ``RemoteTransformer`` c'est d'exposer deux
+fonctions, ``encode`` et ``decode``, et Kinto.js s'assure d'appeler la
+fonction ``encode`` sur chaque enregistrement qui est téléverser dans
+le cloud et d'appeler ``decode`` sur chaque enregistrement
+télécharger. Vous pouvez en lire plus à ce sujet
```

Comment:
Same question as above about "télévers[er / é]" and "télécharg[er / é]" - please ignore me as appropriate. :)